lib/Models/TableCatalogItem.js

```diff
@@ -83,9 +83,9 @@ var TableCatalogItem = function(terria, url, options) {
      * Gets or sets the opacity (alpha) of the data item, where 0.0 is fully transparent and 1.0 is
      * fully opaque.  This property is observable.
      * @type {Number}
-     * @default 0.6
+     * @default 0.8
      */
-    this.opacity = 0.6;
+    this.opacity = 0.8;
```

Comment:
A change of default is technically a breaking change.  We can live with it if there's a good reason, though.